client/src/components/navLink/NavLink.jsx

```diff
@@ -1,7 +1,13 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import classes from './navLink.module.css';
 
-function NavLink() {
-  return <div></div>;
+function NavLink({ text, path, theme }) {
+  return (
+    <Link className={classes.link} data-theme={theme} to={path}>
+      {text}
+    </Link>
+  );
 }
 
 export default NavLink;
```

Comment:
Why have you done separate component for router Link component?

Reply:
because i will use it inside Router to create spa and do not need ordinary link with tag <a>